samigo/pom.xml

```diff
@@ -124,6 +130,27 @@
             </dependency>
         </dependencies>
     </dependencyManagement>
+
```

Comment:
The sakai-kernel-api is used in the samigo-api / SamigoETSProvider and samigo-impl / SamigoETSProviderIMPL via the Event object.

I removed the ones which I could and put them in the appropriate sub-modules. I left lombok in the base pom, as I see it having use in Samigo as a whole.